### PR TITLE
Update branding and documentation

### PR DESCRIPTION
**Pull request by skyarkitekten**
This pull request includes updates to the branding of the project, such as the logo and favicon. It also includes updates to the documentation, replacing references to ISE with Neudesic.

Contains fixes for #15

### CODE REVIEW

**Review by mikeholdorf — APPROVED** [review]
Review and Approved

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
...
 
-The purpose of this repo is to provide guidance to ISE software engineers and data scientists (together forming a Dev Crew) with regards to engineering fundamentals. It describes recommended practices based on continuous project learnings in different areas. It can be used as a tool at the beginning of an engagement that can be shared with customers to help set the project up for success. This repo is *not* intended as a code repo, instead it provides guidance and links to appropriate sample code repos that represent good examples.
+The purpose of this repo is to provide guidance to software engineers, infrastructure engineers, data engineers, and data scientists (together forming a Dev Crew) with regards to engineering fundamentals. It describes recommended practices based on continuous project learnings in different areas. It can be used as a tool at the beginning of an engagement that can be shared with customers to help set the project up for success. This repo is *not* intended as a code repo, instead it provides guidance and links to appropriate sample code repos that represent good examples.
```
Comment (cloudappsguy) [inline CONTRIBUTING.md:5]: First time I'm seeing reference to the term 'dev crew'.  Is this a new definition\term (I get the concept) in the firm?  If we don't have this clearly defined in other spaces/conversations, is there a more Neudesic-terminology-appropriate phrase here?  Dev Squad?  Solution Team?  Other?

File: docs/engineering-feedback/README.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ Even if the feedback has already been raised directly with a product group or on
 
 Capturing and providing high-quality actionable Engineering Feedback is an integral **ongoing** part of all code-with engagements. It is recommended to submit feedback on an ongoing basis instead of batching it up for submission at the end of the engagement.
 
-You should jot down the details of the feedback close to the time when you encounter the specific blockers, challenges, and friction since that is when it is freshest in your mind. The project team can then decide how to prioritize and when to submit the feedback into the official CSE Feedback system (accessible to ISE team members) during each sprint.
+You should jot down the details of the feedback close to the time when you encounter the specific blockers, challenges, and friction since that is when it is freshest in your mind. The project team can then decide how to prioritize and when to submit the feedback into the official CSE Feedback system (accessible to Neudesic Engineering Excellence team members) during each sprint.
```
Comment (cloudappsguy) [inline docs/engineering-feedback/README.md:15]: I don't know what 'submit the feedback into the official CSE Feedback system' means.

**Review by cloudappsguy — APPROVED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (skyarkitekten) [reply CONTRIBUTING.md:5]: It's taken straight from MSFT and we can certainly change it. Any suggestions? 

Added as #19

**Pull request merged**